### DISCUSSION TITLE
Is there a model I can use for seperate drums tracks, ex. kick, snare, hi hats?

### DISCUSSION THREAD

**Discussion by godamn123** [Anjok07/ultimatevocalremovergui#737] [General]
Im not sure if this model exists if anyone could help me would be great thank you.

**Comment by Monitorium100** [comment]
Have the same question.

**Comment by unknownLinguist** [comment]
This would be v cool indeed.

**Comment by paulnth404** [comment]
I imagine it's not far off, music.ai offers it for developers it seems.
https://music.ai/modules/stem-separation/drum-stems/?utm_source=direct&click_section=section_2_modules_list_drum_stems

Edit: Drumsep is a model that does this
https://github.com/inagoy/drumsep

Edit2: You can use DRUMSEP here: https://mvsep.com/en/home

**Comment by Cyan-Chan99** [comment]
same here

**Comment by leocaseiro** [comment]
https://github.com/jarredou/models/releases/tag/DrumSep
